html-forms/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML Forms
+    </title>
+  </head>
+  <body>
+    <h1>
+      Coding Form!
+    </h1>
+    <form action="index.html">
+      <div>
+        Email
+        <br />
+      </div>
+      <label for="email">
```

Comment:
Please put this `label` element around the text for the label, aka "Email" on the line above.